src/core/application.go

```diff
@@ -107,8 +107,15 @@ func (a *Application) StandardRun() {
 		}
 	}
 	// Collect everything
-	worker.RunCollectors()
-
+	changedFiles, diffs := worker.RunCollectors()
+	// Email diffs if changes are detected and email is enabled
+	if len(changedFiles) > 0 && a.config.EmailEnabled {
```

Comment:
Comment above, proposing not having this logic here. Also, note the GitSendEmail already checks `EmailEnabled`